datafusion/core/src/scheduler/mod.rs

```diff
@@ -221,12 +221,12 @@ fn spawn_local_fifo(task: Task) {
 }
 
 #[derive(Debug, Clone)]
-pub struct Spawner {
+pub(crate) struct Spawner {
```

Comment:
This wasn't ever meant to be public, it was accidentally made public as a result of moving from a crate-private module. There was no way to actually construct it, so this is not an API change